django/src/rdwatch/views/site_observation.py

```diff
@@ -97,10 +99,18 @@ def site_observations(request: HttpRequest, pk: int):
     output['images'] = image_serializer.data
     if SatelliteFetching.objects.filter(siteeval=pk).exists():
         retrieved = SatelliteFetching.objects.filter(siteeval=pk).first()
+        celery_data = {}
+        if retrieved.celery_id != '':
```

Comment:
The "pythonic" way to check for empty strings is
```suggestion
        if retrieved.celery_id:
```

Reply:
Yeah, I do this a lot even in JS, mostly because I like being explicit about things and not relying on an implicit conversion in your head when looking through the code.